app/plugin_requests.go

```diff
@@ -42,7 +42,12 @@ func (a *App) ServePluginRequest(w http.ResponseWriter, r *http.Request) {
 
 func (a *App) servePluginRequest(w http.ResponseWriter, r *http.Request, handler func(*plugin.Context, http.ResponseWriter, *http.Request)) {
 	token := ""
-	context := &plugin.Context{}
+	context := &plugin.Context{
```

Comment:
For context, why isn't the session id passed here?

Reply:
It is, check line 87